app/scripts/lib/marketing-email-client.js

```diff
@@ -0,0 +1,63 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/**
+ * A client to talk to the basket marketing email server
+ */
+
+'use strict';
+
+define([
+  'lib/xhr'
+], function (xhr) {
+
+  function MarketingEmailClient(options) {
+    options = options || {};
+
+    var self = this;
+
+    self._xhr = options.xhr || xhr;
+    self._baseUrl = options.baseUrl;
+  }
+
+  MarketingEmailClient.prototype = {
+    _request: function (path, type, accessToken, data, headers) {
+      var url = this._baseUrl + path;
+
+      return this._xhr.oauthAjax({
+        url: url,
+        type: type,
+        accessToken: accessToken,
+        data: data,
+        headers: headers
+      })
+      .fail(function () {
+        // TODO - no server is listening just yet. Remove this when ready.
+        // ignore the error until the server is ready.
+        return {};
+      });
+    },
+
+    fetch: function (accessToken) {
+      return this._request('/lookup-user', 'get', accessToken);
```

Comment:
Ah, nevermind. I was looking at the current API docs. I assume basket's new API won't use that. (related: I noticed the oauth docs [were out of date](https://github.com/mozilla/fxa-oauth-server/commit/07efe68605c78b31c3babffeeb398530c9bb66cf) wrgt the `email` in the /verify response .)